schema_enforcer/ansible_inventory.py

```diff
@@ -1,8 +1,8 @@
 """Ansible Inventory class to generate final hostvars based on group_vars and host_vars."""
-from ansible.inventory.manager import InventoryManager
-from ansible.parsing.dataloader import DataLoader
-from ansible.vars.manager import VariableManager
-from ansible.template import Templar
+from ansible.inventory.manager import InventoryManager  # pylint: disable=import-error
+from ansible.parsing.dataloader import DataLoader  # pylint: disable=import-error
+from ansible.vars.manager import VariableManager  # pylint: disable=import-error
+from ansible.template import Templar  # pylint: disable=import-error
```

Comment:
Why are we running into import errors here now?

Reply:
Because I didn't have ansible installed when i was running the tests
Since it's not a mandatory deps I think it make sense to ignore these errors